embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/opentelemetry/EmbSpanBuilder.kt

```diff
@@ -0,0 +1,70 @@
+package io.embrace.android.embracesdk.opentelemetry
+
+import io.embrace.android.embracesdk.internal.spans.EmbraceSpanBuilder
+import io.embrace.android.embracesdk.internal.spans.SpanService
+import io.opentelemetry.api.common.AttributeKey
+import io.opentelemetry.api.common.Attributes
+import io.opentelemetry.api.trace.Span
+import io.opentelemetry.api.trace.SpanBuilder
+import io.opentelemetry.api.trace.SpanContext
+import io.opentelemetry.api.trace.SpanKind
+import io.opentelemetry.context.Context
+import io.opentelemetry.sdk.common.Clock
+import java.util.concurrent.TimeUnit
+
+internal class EmbSpanBuilder(
+    private val embraceSpanBuilder: EmbraceSpanBuilder,
```

Comment:
I'm not a massive fan of naming two classes so similarly. Can one (or both) be renamed to make them easier to distinguish?